content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,105 @@
+@Video(https://www.youtube.com/watch?v=Qle9Y1KdD2Y)
+This allows users to prepare their tracks on a laptop using Mixxx and export them to a USB stick for use on a standalone controller at their gig.
+
+### Saved Loops
+
+Mixxx 2.4 adds support for Saved Loops, which can be assigned to any Hot Cue slot.
+If you already have tracks from Serato DJ with Saved Loops, Mixxx 2.4 will import them automatically.
+
+@Video(https://www.youtube.com/watch?v=f8hI_uv9zHY)
+
+### Effect Chains
+
+Mixxx 2.4 introduces Effect Chains, a set of effects which can be saved and reloaded with all their parameters and individual Meta Knob mappings.
+Effect Chains can also be exported and imported, which allows you to share them with other Mixxx DJs.
+Once you've created an Effect Chain, you can create a custom Super Knob mapping which turns the chain into a Quick-Effect, where you take creative control with a single knob.
+Effect parameters can be reordered and or hidden.
+In the effect units, Mixxx 2.4 also displays the value and unit of effect knob parameters while they are being adjusted.
+
+### Track Menu Enhancements
+
+The track menu in Mixxx 2.4 is enhanced by several new features:
+
+- Select loaded track in library.
+- Analyze/re-analyze with variable/constant BPM on a per-track basis.
+- Look up track metadata at Discogs, SoundCloud and LastFM.
+- Remove track files from disk.
+
+### Sync Lock Improvements
+
+Mixxx 2.4 adds support for manually selecting the Sync Leader by clicking on the new button with the crown symbol next to the Sync button.
+This mode is useful when tracks have variable BPM.
+All decks with sync enabled will follow the changing BPM of the Sync Leader.
+@Video(https://www.youtube.com/watch?v=REiG8DBKr-M)
+
+### Many other changes
+
+##### Soundstretcher / Key-Shift Algorithms
+
+Mixxx 2.4 now supports three different key shift algorithms:
+
+- Soundtouch (Basic audio quality - medium CPU usage)
+- Rubberband R2 (Good audio quality - higher CPU usage)
+- Rubberband R3 (High audio quality - very high CPU usage)
+
+##### Major rework of the Waveform Rendering code
+
+The waveform display has been rewritten, solving some longstanding performance issues, especially on macOS.
+
+##### Native macOS ARM builds for Apple silicon
+
+##### Controller Backend
+
+Modernized Javascript ES7 based controller mapping system.
+HID backend rework.
+
+## Mixxx 2.4 Factsheet
+
+##### Supported platforms:
+
+- Windows 7 or later on 64bit processors
+- macOS 10.12 or later on Intel processors
+- macOS 11.0 or later on ARM processors (Apple Silicon)
+- Ubuntu Linux 20.04 or later (other Linux distributions are supported by third parties)
+
+##### Supported controllers:
+
+- Mappings for 129 MIDI controllers included
+- Mappings for 16 HID controllers included
+
+See the [full list](https://manual.mixxx.org/2.4/hardware/manuals#controllers) of supported controllers in the Mixxx Manual.
+
+## Media information
```

Comment:
```suggestion
## Press Information
```

Reply:
The term "Press Information" has another meaning, which does not fit here. A "Press Information" is about specific topic to announce. The whole article could be called a "Press Information", but this is the heading of the chapter with general information. For this broader scope "Media information" is the correct wording.

Reply:
This release announcement is not specifically targeted towards the press, it's also for Mixxx users, package maintainers and other interested parties.

This section is specifically targeted to DJ news channels, blogs, etc. and provides download links, image packs, logo and a press contact. Maybe you are confusing it with "news release"/press handout" (🇩🇪 "Pressemitteilung")?

My impression is that "media" instead of "press" was used here because "press" traditionally referred to printed magazines and newspapers and nowadays we also have blogs and social media. But today it is common to use "press" to mean those channels too and helps these people to spot the info easier. Hence my suggestion.

As mixxx is a "media player", the current headline is not that descriptive and without looking at the actual content the reader does not know whether it will give information tailored to journalists or if it lists "media information" like supported audio formats.

Reply:
as a native speaker, it's not a big deal to use one or the other.  "media" is not super confusing in the context, neither is "press"